integration-tests/utils.py

```diff
@@ -319,9 +319,15 @@ def asyncify(f):
                         break
                     await asyncio.sleep(3)
                 else:
-                    raise TimeoutError(
-                        'Unable to find blocked status on unit {0} - {1} {2}'.format(
-                            unit.name, unit.workload_status, unit.agent_status))
+                    # We waited for the block msg but we did not see one.
+                    # See https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/545
+                    # and https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/504
+                    # We will have to raise but not until the above are addressed
```

Comment:
Wait, wait. The known issue is that we sometimes see a blocked message when we're not supposed to. This sounds like the opposite - we sometimes _don't_ see the blocked message when we _are_ supposed to. I would like to look into why that's happening.

Reply:
I looked into this and opened a new issue: https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/596. I would like to fix this in the kubernetes-worker charm instead of working around it in the test here. Any objections?

Reply:
+1 .  This PR was just a temporary workaround to unblock the tests.